implementations/rust/ockam/ockam_command/src/kafka/producer/delete.rs

```diff
@@ -0,0 +1,52 @@
+use crate::node::{get_node_name, initialize_node_if_default};
+use crate::util::{node_rpc, parse_node_name, Rpc};
+use crate::{docs, fmt_ok, node::NodeOpts, CommandGlobalOpts};
+use clap::Args;
+use colorful::Colorful;
+use ockam_api::nodes::models;
+use ockam_core::api::Request;
+
+const AFTER_LONG_HELP: &str = include_str!("./static/delete/after_long_help.txt");
```

Comment:
I wonder if we could avoid mistakes by having a macro that is even smarter and:

 - takes the name of the current module to build the path
 - errors out (at compile time) if the path does not exist

Reply:
I like that idea! It's definitely worth it, I've lost the count of how many times I've imported the wrong file. I'll open an issue for this.